go.mod

```diff
@@ -11,6 +11,7 @@ require (
 	github.com/hashicorp/go-retryablehttp v0.7.1
 	github.com/hashicorp/terraform-plugin-log v0.4.0
 	github.com/hashicorp/terraform-plugin-sdk/v2 v2.17.0
+	github.com/robfig/cron v1.2.0
```

Comment:
Could we use v3 as suggested here: import "github.com/robfig/cron/v3"
https://github.com/robfig/cron#cron

Reply:
updated